projects/js-packages/ai-client/changelog/update-ai-client-expose-compiles-files

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+AI Client: expose build folder
```

Comment:
Maybe something like this would be more descriptive?
```suggestion
AI Client: include built JavaScript code in addition to TypeScript.
```

Technically there wasn't really a "build" folder before to expose, since even though a build script existed in `package.json` it wasn't being run (and was broken). 😀

Reply:
yes! much better